buildSrc/src/main/java/org/elasticsearch/gradle/ElasticsearchDistribution.java

```diff
@@ -270,6 +276,11 @@ void finalizeValues() {
                     "platform not allowed for elasticsearch distribution [" + name + "] of type [" + getType() + "]"
                 );
             }
+            if (getType() == Type.DOCKER && bundledJdk.isPresent()) {
+                throw new IllegalArgumentException(
+                    "bundledJdk not allowed for elasticsearch distribution [" + name + "] of type [docker]"
```

Comment:
bundledJdk *setting*?

Reply:
or property or something like that? as it is written it sounds like docker can't use the bundledJdk, when in face it only uses it.